test/validators/transfer.ts

```diff
@@ -67,18 +53,6 @@ describe('Transfer Validator Tests', () => {
     await helpers.transferValidator(4, clusterResult1.validators[0].publicKey, helpers.DataGenerator.cluster.byId(clusterId), `${minDepositAmount / 4 * 7 * 2}`, [GasGroup.TRANSFER_VALIDATOR_NON_EXISTING_POD]);
   });
 
-  it('Transfer validator with not enough amount', async () => {
-    // Register validator
-    const { clusterId } = await helpers.registerValidators(4, 1, minDepositAmount, [1, 2, 3, 9]);
-
-    // Transfer to cluster with not enough amount
-    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).transferValidator(
-      clusterResult1.validators[0].publicKey,
-      (await helpers.registerPodAndDeposit(4, helpers.DataGenerator.cluster.byId(clusterId), helpers.CONFIG.minimalOperatorFee)).clusterId,
-      helpers.DataGenerator.shares(helpers.DB.validators.length),
-    )).to.be.revertedWith('NotEnoughBalance');
-  });
-
   // GOING ABOVE GAS LIMIT
   it('Transfer validator to an existing pod', async () => {
```

Comment:
Below are tests:
Transfer validator with removed operator in old cluster A-Z
Transfer validator with removed operator in new cluster A-Z
Transfer validator with removed operator in old cluster Z-A
Transfer validator with removed operator in new cluster Z-A
To
Transfer validator with a removed operator into an existing pod
Transfer validator with a removed operator into an empty pod
Transfer validator with a removed operator into an existing pod
Transfer validator with a removed operator into an empty pod